### DISCUSSION TITLE
3.5 Lookup Tables Error.

### DISCUSSION THREAD

**Discussion by LastChalice** [Andrewthe13th/Inventory_Kamera#437] [Q&A] [answered]
Hey everyone. 
Trying to update Inventory Kamera with new 3.5 genshin data, and i keep on encountering these two weird errors.

`2023-03-03 17:12:13.438|ERROR|DatabaseManager|An error was encountered when gathering information for the character HtmlAgilityPack.HtmlNode
2023-03-03 17:12:13.438|ERROR|DatabaseManager|System.NullReferenceException: Object reference not set to an instance of an object.
   at InventoryKamera.DatabaseManager.<>c__DisplayClass32_0.<UpdateCharacters>b__2(List`1 entry)`

Any idea what this could be about?
Attaching both logs here.
[InventoryKamera.debug.log](https://github.com/Andrewthe13th/Inventory_Kamera/files/10886640/InventoryKamera.debug.log)
[InventoryKamera.log](https://github.com/Andrewthe13th/Inventory_Kamera/files/10886641/InventoryKamera.log)

Edit: Apparently can't scan right now because of that.

**Comment by Cupcak3** [comment, marked as answer]
The log indicates you are using v1.3.7 which is out of date. v1.3.8 contains a fix for this issue

**Reply by LastChalice** [reply]
Thanks.

**Reply by LastChalice** [reply]
Any way to update without reinstalling?
